vision/samples/v1/samplegen_basics.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "samplegen_basics")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-vision
+
+# sample-metadata
+#   title: This is the sample title
+#   description: This is the sample description
+#   usage: python3 samples/v1/samplegen_basics.py [--display_name "This is the default value of the display_name request field"]
+
+# [START samplegen_basics]
```

Comment:
@tswast You mentioned a preference for **not showing the method definition** in the embedded code? (START/END)

Do you know if that's the **standard nowadays?** I've seen both types (_with and without method signature_).

> No opinion on my end! The template for samples can be updated to whatever is standard! 😄 

**Does anyone have a canonical example?** e.g. for Node.js they have [these canonical examples](https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head#canonical-samples)

I'd love to update the structure **if** that's what's needed. Would it be a blocker if we show the method definition?

> /cc @busunkim96 – This is the biggest piece of feedback from the previous PR ^

Reply:
Python never settled on the canonical format, but there are a few of us working towards doing so (the original PR was currently [here](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/1756), and we have an AI on cleaning it up and creating a sample style guide).

Personally, I have a slight preference to leaving the function signature in (since it would make it easier for me to copy/paste into existing code).

Reply:
Thanks for that context @kurtisvg!

I'm not biased in either direction, iirc we didn't originally have the method signature and then we added it to the generator template.

**My preference: _"leave the function signature in"_ +1**

Happy to change this at any time!

I would find it most helpful if we can say that it's OK to launch in Q1 with the method signature and we can change after (_ie. let's not block on this one item_)

We _should_ block on things like fixing the docstring format, though. Which we'll fix!

Reply:
+1 - this is a nit for me.